apps/server/src/modules/auth/guard.ts

```diff
@@ -92,8 +92,10 @@ class AuthGuard implements CanActivate {
         return false;
       }
 
-      // @ts-expect-error body is user here
-      req.user = body.user;
+      req.user = await this.prisma.user.findUnique({
```

Comment:
duplicate database query here

Reply:
user session was filter by next-auth, we dont has enough information